### PR TITLE
CX-613: dependancy fix for 1.27.x 

### PR DESCRIPTION
**Pull request by Johnbastian**
- Fixing dependencies for 1.27.x
- Releasing 1.27.2
- Merging into a release branch

### CODE REVIEW

**Review by Ryanbarker0 — APPROVED** [review]
LGTM

**Review by jonurry — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -22,12 +22,12 @@
   "homepage": "https://github.com/trayio/falafel#readme",
   "dependencies": {
     "@trayio/threadneedle": "~1.12.1",
-    "aws-sdk": "~2.413.0",
+    "aws-sdk": "~2.1074.0",
```
Comment (ssong) [inline package.json:25]: 👀  whoa

**Review by EdinaBolkeny — APPROVED** [review]
LGTM

**Review by ssong — APPROVED** [review]

**Pull request merged**